rspace/src/main/scala/coop/rchain/rspace/history/package.scala

```diff
@@ -50,16 +57,16 @@ package object history {
       }
     }
 
-  def lookup[T, K, V](store: ITrieStore[T, K, V], branch: Branch, key: K)(
-      implicit codecK: Codec[K]): Option[V] = {
+  def lookup[T, K, V](branch: Branch, key: K)(implicit store: ITrieStore[T, K, V],
```

Comment:
Could we put a newline after `implicit`?  `scalafmt` is usually cool with that and aids readability IMHO.

Reply:
After implicit?

Reply:
I'm afraid `scalafmt` moves it back here, we'd have to change the rules

Reply:
I think I got it right this time